test/integration/checkout_int_test.go

```diff
@@ -71,6 +71,30 @@ func (suite *CheckoutIntegrationTestSuite) TestCheckout() {
 	})
 }
 
+func (suite *CheckoutIntegrationTestSuite) TestCheckoutPerl() {
+	suite.OnlyRunForTags(tagsuite.Checkout, tagsuite.Critical)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	// Checkout and verify.
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/Perl-Alternative", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Checking out project: ")
+	cp.Expect("Setting up the following dependencies:")
+	cp.Expect("All dependencies have been installed and verified", e2e.RuntimeSourcingTimeoutOpt)
+	cp.Expect("Checked out project")
```

Comment:
We also want to `cp.ExpectExitCode(0)`, right?

Reply:
Yep, thanks for spotting!